newsroom/web/default_settings.py

```diff
@@ -344,6 +344,7 @@
     "item_actions": {},
     "display_abstract": DISPLAY_ABSTRACT,
     "display_credits": False,
+    "view_content_tooltip_email": "test@localhost.com",
```

Comment:
not sure if it makes sense to have some default here, would probably rather hide it if it's not configured